config/jobs/kubernetes/sig-aws/eks/k8s-aws-eks-presubmits.yaml

```diff
@@ -0,0 +1,33 @@
+presubmits:
+  # Run Kubernetes 1.10 branch e2e tests with EKS prod build 1.10
+  # run conformance tests
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-1-11-aws-eks-1-11-prod-conformance
```

Comment:
Should we be mentioning a trigger command somewhere below?

Reply:
`/test pull-kubernetes-e2e-1-11-aws-eks-1-11-prod-conformance` is automatically populated now